packages/@aws-cdk/aws-rds/lib/perms.ts

```diff
@@ -0,0 +1,11 @@
+export const DATA_API_ACTIONS = [
```

Comment:
Always nice to link to docs where policies/actions come from so we can verify them or see when there have been updates.

```suggestion
// https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/data-api.html#data-api.access
export const DATA_API_ACTIONS = [
```